opensource/project/set-up-dev-env.md

```diff
@@ -92,7 +92,7 @@ can take over 15 minutes to complete.
 
 1. Open a terminal.
 
-   For Mac users, use `docker-machine status your_vm_name` to make sure your VM is running. You
+   For <a href="/toolbox/overview/" target="_blank">Docker Toolbox</a> users, use `docker-machine status your_vm_name` to make sure your VM is running. You
```

Comment:
Sure thing.  I started to do that but noticed the next link in the page was using the HTML-style link so I just followed that.  

If you like, I can make that update as well.  I assume you'd want that as a separate PR, but should I open an issue to make that update as well?